src/main/java/org/aion/wallet/ui/components/HistoryPane.java

```diff
@@ -0,0 +1,83 @@
+package org.aion.wallet.ui.components;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import org.aion.wallet.connector.BlockchainConnector;
+import org.aion.wallet.connector.WalletBlockchainConnector;
+import org.aion.wallet.connector.dto.TransactionDTO;
+import org.aion.wallet.util.BalanceFormatter;
+
+import java.net.URL;
+import java.util.List;
+import java.util.ResourceBundle;
+import java.util.stream.Collectors;
+
+public class HistoryPane implements Initializable {
+
+    private final BlockchainConnector blockchainConnector = new WalletBlockchainConnector();
+    @FXML
+    private TableView<TxRow> txListOverview;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        TableColumn<TxRow, String> typeCol = new TableColumn<>("Type");
+        TableColumn<TxRow, String> addrCol = new TableColumn<>("Addr");
+        TableColumn<TxRow, String> valueCol = new TableColumn<>("Value");
+        typeCol.setCellValueFactory(new PropertyValueFactory<>("type"));
+        addrCol.setCellValueFactory(new PropertyValueFactory<>("address"));
+        valueCol.setCellValueFactory(new PropertyValueFactory<>("value"));
+
+        txListOverview.getColumns().addAll(typeCol, addrCol, valueCol);
+        reloadHistory();
+    }
+
+    private void reloadHistory() {
```

Comment:
I think that for this demo version we call this when we register for a HeaderPaneButtonEvent